mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedGeneralTypeParams.scala

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
@dbtsai Sorry for the long delay. I'm not so sure of your mean by "integrate into the other part of Params", what's it for?